### PR TITLE
Andreas/upgrades

### PR DESCRIPTION
**Pull request by abigger87**

### CODE REVIEW

File: tests/api/auth.test.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ describe("Auth Tests", () => {
     return request
       .get(devLoginUrl)
       .query({ id: "a", role: "admin" })
-      .expect(200, { ...user, ...addedVals });
+      .expect(200, { ...user, ...addedVals })
+      .catch(console.log);
```
Comment (MaxLeiter) [inline tests/api/auth.test.js:20]:
```
.catch((e) => console.error(e))
```

File: dist/components/Blob.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from "react";
+
+import styled from "styled-components";
+
+import * as THREE from "three";
+import SimplexNoise from "simplex-noise";
+
+const Blob = () => {
+  useEffect(() => {
+    let $canvas = document.querySelector("#blob canvas"),
```
Comment (MaxLeiter) [inline dist/components/Blob.tsx:10]: we should probably use a ref versus querySelector? querySelector requires walking the DOM tree.

File: dist/components/Blob.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+    let lightBottom = new THREE.DirectionalLight(0xfeda22, 0.8);
+    lightBottom.position.set(0, -500, 400);
+    lightBottom.castShadow = true;
+    scene.add(lightBottom);
+
+    let ambientLight = new THREE.AmbientLight(0x86dcea);
+    scene.add(ambientLight);
+
+    let sphere: any = new THREE.Mesh(geometry, material);
```
Comment (MaxLeiter) [inline dist/components/Blob.tsx:48]: You can use https://github.com/react-spring/react-three-fiber for types and avoid using `any` in this file

File: tests/api/person.test.js [hunk 4]
```diff
@@ -15,6 +15,7 @@ describe("Peoples", () => {
       .expect(200)
       .then(res => {
         expect(res.body.person.Profile).toBeDefined();
-      });
+      })
+      .catch(console.log);
```
Comment (MaxLeiter) [inline tests/api/person.test.js:19]: .catch issue here too

File: pages/api-directory.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import * as Sentry from "@sentry/browser";
+import "react-loader-spinner/dist/loader/css/react-spinner-loader.css";
```
Comment (MaxLeiter) [inline pages/api-directory.tsx:3]: Let's get rid of react-loader-spinner if we can. Newer versions of next.js don't allow global CSS imports so we'd need to apply this to every page or do some hacks.

Comment (MaxLeiter) [reply pages/api-directory.tsx:3]: (doesn't need to happen now just a note for future)

Comment (abigger87) [reply pages/api-directory.tsx:3]: good look

**Pull request merged**